db/migrate/20250731144632_add_scoped_index_to_starting_invoice_subscription.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddScopedIndexToStartingInvoiceSubscription < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
+
+  def change
+    add_index :invoice_subscriptions,
```

Comment:
it's not really needed but as a safety measure we can add
`if_not_exists: true`, so this index will only be added if it does not exist yet